### PR TITLE
Kube mgr

### PR DESCRIPTION
**Pull request by davidhadas**
This PR adds KubeMgr - a go package that interacts with KubeAPI to get and set Guardian CRDs and ConfigMaps

Guardians can be get by guard-gate instances (inside queue-proxy), by the guard-learner (guard-service) and by the guard-ui (for manual visualization and control of Guardians).

Guardians can be set by the guard-learner (guard-service) and by the guard-ui.

### CODE REVIEW

**Comment by davidhadas** [comment]
pls merge #45  first

File: pkg/guard-kubemgr/guard-kubemgr.go [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+		return k.setCm(ns, sid, guardianSpec)
+	} else {
+		return k.setCrd(ns, sid, guardianSpec)
+	}
+}
+
+// GetGuardian - Returns a Guardian that was read from Crd or from ConfigMap or an auto-activated Guardian
+// Never returns nil
+//    ns is the namespace being used
+//    sid is teh service identifier being used
```
Comment (psschwei) [inline pkg/guard-kubemgr/guard-kubemgr.go:348]:
```suggestion
//    sid is the service identifier being used
```

**Review by psschwei — COMMENTED** [review]
go files also need the standard boilerplate

**Review by maximilien — APPROVED** [review]
/approve
/lgtm

**Pull request merged**